docs/docs/user-guide/incidents/forms.mdx

```diff
@@ -47,3 +47,21 @@ While a form is being completed, the user can **Cancel** to discard any changes,
 ## Attorney review
 
 A new tab on the left "Forms" lists all of the _Draft_ and _Completed_ forms for leadership and attorney review. For each form, users can view/edit, delete, and a special **Attorney Review** option. This option shows relevant incident details and the values filled out in the form. It also provides an attorney status dropdown and two new fields for attorney notes and open questions.
+
+## Additional attorney questions
+
+If additional questions are required for the attorney team to assess the information on the form, each form type has the option of asking additional attorney questions. This schema follows the same format as the main form schema. The additional attorney questions will appear only on the Attorney Section when performing the Attorney Review from the Forms page.
+
+![](/img/attorney-section.png)
+
+## Scoring schema
```

Comment:
If it was my first time encountering this, I would wonder where the risk score goes and what can I do with it? Maybe a completed example/usage would be helpful to address that.

Reply:
Thanks for the feedback. I added an example.